### ISSUE TITLE
Implement directus to feedbackCard

### ISSUE THREAD

**Issue by Jonroi** [Alt-Org/Altzone-WebPages#307]
# ✨ **Proposal: Integrate Directus with FeedbackCard Component**

---

## 🌟 **Overview**

This proposal outlines a plan to integrate Directus with the `FeedbackCard` component to enhance functionality, streamline data management, and improve the overall user experience. By leveraging Directus as a backend, the `FeedbackCard` component will dynamically manage and display user feedback with ease and scalability.

---

### **Enhanced User Experience**
- Enable real-time feedback updates directly in the `FeedbackCard`.

### **Operational Advantages**
- Simplify content management using Directus's intuitive interface.
- Minimize development overhead by centralizing feedback data management.

---

### **1. Directus Configuration**
- **Setup Collections**: Create a `Feedback` collection with the following fields:
  - `Name`: String, required.
  - `feedbackText`: Text, required.
  - `rating`: Integer, optional.
  - `timestamp`: DateTime, auto-generated.
- **Permissions**: Configure roles and permissions to secure feedback data access.

### **2. Backend Integration**
- **Directus API**: Use Directus’s REST. with Directus SDK

---

## 🎯 **Technical Considerations**

- **Security**:
  - Secure API endpoints with appropriate authentication and authorization (if needed).
  - Validate all input data to maintain data integrity.
---

## 📎 **References & Resources**

- [Directus Documentation](https://docs.directus.io/)  
- [Directus feed back with vue](https://docs.directus.io/blog/building-user-feedback-widget-with-vuejs-directus.html)  

---

**Comment by leolabdev** [comment]
@SProkopios @Jonroi lets fix first that one : https://github.com/Alt-Org/Altzone-WebPages/pull/313?

**Comment by Jonroi** [comment]
Yes this was the plan but I didn't see those fixes you mentioned earlier... Still learning 🙄
